payments-hook/src/useCardNumbers/useCardNumbers.ts

```diff
@@ -71,21 +82,144 @@ export default function useCardNumbers(userCardNumbers = INITIAL_CARD_NUMBER) {
   };
 }
 
-function getCardNumbersError(input: string) {
+export type CardNetWorks = 'VISA' | 'MASTER' | 'AMEX' | 'DINERS' | 'UNIONPAY';
+
+type getCardNumbersErrorProps = {
+  input: string;
+  target: 'FIRST' | 'SECOND' | 'THIRD' | 'FOURTH';
```

Comment:
아래와 같이 타입 재사용도 가능해 보입니다!
```suggestion
  target: CardNumbersKeys;
```

Reply:
https://github.com/woowacourse/react-modules/pull/135/commits/ecc9ae85401c1e4494ed610b70fe149621461719

CardNumbersKeys를 적용하였습니다!